### PR TITLE
fix: cloned volume could not be recognized on Windows node when source volume is mounted on the same node

### PR DESCRIPTION
**Pull request by andyzhangx**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
fix: ensure that readonly is disabled when disks are brought online
This PR could fix the cloned volume not recognized on the same Windows node (when source volume is also mounted on the same node)

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
fix: ensure that readonly is disabled when disks are brought online
```

### CODE REVIEW

File: test/e2e/suite_test.go [hunk 1]
```diff
@@ -346,6 +347,9 @@ func convertToPowershellorCmdCommandIfNecessary(command string) string {
 		return "echo 'overwrite' | Out-File -FilePath C:\\mnt\\test-1\\data.txt; Start-Sleep 3600"
 	case "grep 'hello world' /mnt/test-1/data":
 		return "Get-Content C:\\mnt\\test-1\\data.txt | findstr 'hello world'"
+	case "df -h | grep /mnt/test- | awk '{print $2}' | grep 20.0G":
+		// todo: find a way to get the size of the volume on Windows node
```
Comment (lippertmarkus) [inline test/e2e/suite_test.go:351]: `fsutil volume diskfree C:\\mnt\\test-1\\ | Select-String "Total bytes" | Select-String "15.0 GB"`

**Review by lippertmarkus — CHANGES_REQUESTED** [review]

File: test/e2e/suite_test.go (hunk 1 above)
Comment (andyzhangx) [reply test/e2e/suite_test.go]: thanks, fixed.

**Pull request merged**